server/lib/nl/common/utils.py

```diff
@@ -121,20 +125,55 @@ def sv_existence_for_places_check_single_point(
 
   existing_svs = {}
   existsv2places = {}
+  facet_info = series_facet.get('facets', {})
+  date_to_check: types.Date = None
+  date_string = None
+  # For now, only check date if date is a single date.
+  # TODO: check for a date range.
+  if is_single_date(date):
+    date_to_check = date
+    date_string = date.get_date_string()
```

Comment:
I feel this should happen in the caller?  The is_single_date is sprinkled everywhere.  wdyt of making it part of Classification?  So, pass the date to this function only if that classification exists.

Reply:
done